var/spack/repos/builtin/packages/py-mg-rast-tools/package.py

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyMgRastTools(PythonPackage):
+    """Repository of scripts and libraries for using the MG-RAST API and
+    MG-RAST data."""
+
+    homepage = "https://github.com/MG-RAST/MG-RAST-Tools"
+
+    version('2018.04.17', git='https://github.com/MG-RAST/MG-RAST-Tools.git', commit='a40c6e6539ad0bc1c08e1b03dfc0a9759755a326')
+
+    depends_on('perl', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-prettytable')
+    depends_on('py-poster')
+    depends_on('py-requests')
+    depends_on('py-requests-toolbelt')
+    depends_on('py-scipy')
+    depends_on('py-numpy')
+    depends_on('r-matr')
+    depends_on('shocklibs')
+    depends_on('perl-list-moreutils')
+    depends_on('perl-exporter-tiny')
+    depends_on('perl-libwww-perl')
+    depends_on('perl-http-message')
+    depends_on('perl-json')
```

Comment:
All of these Python/Perl/R dependencies should likely be `type=('build', 'run')`.